ddl/placement/bundle.go

```diff
@@ -167,60 +151,31 @@ func NewBundleFromSugarOptions(options *model.PlacementSettings) (*Bundle, error
 	}
 	schedule := options.Schedule
 
-	var constraints Constraints
-	var err error
+	// regions must include the primary
+	sort.Strings(regions)
+	primaryIndex := sort.SearchStrings(regions, primaryRegion)
+	if primaryIndex >= len(regions) || regions[primaryIndex] != primaryRegion {
+		return nil, fmt.Errorf("%w: primary region must be included in regions", ErrInvalidPlacementOptions)
+	}
+
+	var Rules []*Rule
 
-	Rules := []*Rule{}
 	switch strings.ToLower(schedule) {
 	case "", "even":
```

Comment:
`majority_in_primary` may be a better default value because it is more widely used, I think. 

Another concern is that we can move this default assignment out of this function to the construction of `model.PlacementSettings` . In this way we can always get an explicitly configured `SCHEDULE` option when querying by `show create placement policy` . This behavior will be more close to other options like `ENGINE` or `CHARSET`

Reply:
It can be done later, recorded into the tracking issue.

Reply:
make sense with @wangchao